test/e2e/gc/testdata/env/config-features.patch

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-features
+  namespace: ${SYSTEM_NAMESPACE}
+data:
+  responsive-revision-gc: enabled
```

Comment:
This would add more complexity to getting the test setup/running - given how the runner and test binary are separate processes/binaries. I think this can be explored after the fact.

Do you have ideas on how you'd do it?

Reply:
I opened an upstream issue for this https://github.com/golang/go/issues/44929